app/components/ny_times/client.rb

```diff
@@ -3,29 +3,14 @@
 module NyTimes
   # HTTParty client for the NewYork Times that takes an article path
   # and get's the text of the article
-  class Client
-    attr_reader :code, :response_body
-    include HTTParty
-    base_uri 'nytimes.com'
+  class Client < GenericClient
```

Comment:
Assumes too much for instance variable '@response_body' (InstanceVariableAssumption)